src/PyNvCodec/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright 2019 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+cmake_minimum_required(VERSION 3.20)
+
+project(PyNvCodec VERSION 1.0)
+
+set(PYNVCODEC_VERSION_MAJOR 1)
+set(PYNVCODEC_VERSION_MINOR 0)
+
+configure_file("inc/Version.hpp.in" "pynvcode_version.h")
+
+find_package(Python3 3.6 REQUIRED COMPONENTS Interpreter Development)
+
+option(FETCHCONTENT_QUIET OFF)
+include(FetchContent)
+fetchcontent_declare(
+    pybind11
+    URL https://github.com/pybind/pybind11/archive/v2.10.0.tar.gz
+    URL_HASH
+        SHA256=eacf582fa8f696227988d08cfc46121770823839fe9e301a20fbce67e7cd70ec
+    DOWNLOAD_EXTRACT_TIMESTAMP
```

Comment:
Would you be fine with ommiting this line ? It requires a very recent version of CMake. At least my default 3.22 on Ubuntu 2004 was no able to use it.

Reply:
this was the more obvious syntax to silence at CMake policy. So I have to google which policy this was and what was the portable way to set it

Reply:
I see :D so subject to possibly change. My cmake interprets this as part of the hash but i guess we can also find a way to omit this.

Reply:
Fixed now. With CMP0137